### PR TITLE
fix(database-container): fix error when creating project with no database

### PR DESCRIPTION
**Pull request by gmq**

### CODE REVIEW

File: lib/potassium/recipes/database_container.rb [hunk 1]
```diff
@@ -28,11 +28,12 @@ class Recipes::DatabaseContainer < Rails::AppBuilder
     YAML
 
   def create
-    copy_file '../assets/docker-compose.yml', 'docker-compose.yml'
+    db_type = get(:database)
+    return if db_type == :none
```
Comment (rjherrera) [inline lib/potassium/recipes/database_container.rb]: I think this should be as in the frontend recipe, as in some cases the value is `:None` and in others `:none`.

```ruby
return if [:None, :none].include? db_type.to_sym
```

Comment (ldlsegovia) [reply lib/potassium/recipes/database_container.rb]: o `db_type.downcase.to_sym`

Comment (blackjid) [reply lib/potassium/recipes/database_container.rb]: @rjherrera that method `get()` is only used for getting results from the args and the questions form the "ui"??? if that is so, we could add some functionality to the `get` method to return nil whenever is `:None` or `:none`, something like that.

Comment (rjherrera) [reply lib/potassium/recipes/database_container.rb]: I think so, I could test it but I remember having problems with recipes that didn't cover the `:None` case. Your suggestion seems to be a nice solution!

**Pull request merged**